pom.xml

```diff
@@ -126,6 +127,11 @@
             <artifactId>httpclient</artifactId>
             <version>4.5.8</version>
         </dependency>
+        <dependency>
+            <groupId>org.mapstruct</groupId>
```

Comment:
This lib is 'approved' by CH and allows us to easily map from entity classes to DTOs, essentially without having to write any (well, much) code.